app/modules/profile/routes.py

```diff
@@ -53,3 +54,35 @@ def my_profile():
         pagination=user_datasets_pagination,
         total_datasets=total_datasets_count
     )
+
+# Allows to go to the profile of the user selected
+
+
+@profile_bp.route('/profile/<int:user_id>/', methods=["GET"])
+def user_profile(user_id):
+    page = request.args.get('page', 1, type=int)
+    per_page = 5
+
+    # Buscar el usuario según el user_id proporcionado
+    user = db.session.query(User).get(user_id)
+    if not user:
+        abort(404)  # Retorna un error 404 si el usuario no existe
+
+    # Obtener los datasets del usuario
+    user_datasets_pagination = db.session.query(DataSet) \
+        .filter(DataSet.user_id == user_id) \
+        .order_by(DataSet.created_at.desc()) \
+        .paginate(page=page, per_page=per_page, error_out=False)
+
+    total_datasets_count = db.session.query(DataSet) \
+        .filter(DataSet.user_id == user_id) \
+        .count()
+
+    return render_template(
+        'profile/summary.html',
```

Comment:
This template is missing from the PR!

Reply:
Was not very sured wdym but indeed there was an error surely from the commits issues i had earlier, this is already fixed.